File: My Schedule/My Schedule/MainForm.rb
require "mscorlib"
require "System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089"
require "System.Drawing, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"

class MainForm < System::Windows::Forms::Form
	def initialize()
		self.InitializeComponent()
	end

	def InitializeComponent()
		@label1 = System::Windows::Forms::Label.new()
		@button1 = System::Windows::Forms::Button.new()
		@button2 = System::Windows::Forms::Button.new()
		self.SuspendLayout()
		# 
		# label1
		# 
		@label1.BackColor = System::Drawing::Color.White
		@label1.ForeColor = System::Drawing::SystemColors.ControlText
		@label1.Location = System::Drawing::Point.new(51, 26)
		@label1.Name = "label1"
		@label1.Size = System::Drawing::Size.new(156, 108)
		@label1.TabIndex = 0
		# 
		# button1
		# 
		@button1.BackColor = System::Drawing::Color.ForestGreen
		@button1.ForeColor = System::Drawing::SystemColors.ButtonHighlight
		@button1.Location = System::Drawing::Point.new(235, 25)
		@button1.Name = "button1"
		@button1.Size = System::Drawing::Size.new(99, 45)
		@button1.TabIndex = 1
		@button1.Text = "Show Schedule"
		@button1.UseVisualStyleBackColor = false
		@button1.Click { |sender, e| self.Button1Click(sender, e) }
		# 
		# button2
		# 
		@button2.BackColor = System::Drawing::Color.Crimson
		@button2.ForeColor = System::Drawing::SystemColors.ButtonHighlight
		@button2.Location = System::Drawing::Point.new(235, 76)
		@button2.Name = "button2"
		@button2.Size = System::Drawing::Size.new(99, 44)
		@button2.TabIndex = 2
		@button2.Text = "Clear Schedule "
		@button2.UseVisualStyleBackColor = false
		@button2.Click { |sender, e| self.Button2Click(sender, e) }
		# 
		# MainForm
		# 
		self.BackColor = System::Drawing::SystemColors.ActiveCaptionText
		self.ClientSize = System::Drawing::Size.new(548, 331)
		self.Controls.Add(@button2)
		self.Controls.Add(@button1)
		self.Controls.Add(@label1)
		self.Name = "MainForm"
		self.Text = "My Schedule"
		self.ResumeLayout(false)
	end

	def Button1Click(sender, e)
		@label1.Text = "Period 1 - Studyhall\nPeriod 2 -Photography\nPeriod 3 - Sci-fi & Fantasy\nPeriod 4 - TC Speech\nPeriod 5 - Intergrated Algebra\nPeriod 6 - Programming 1\nPeriod 7 - AP Physics 2\nPeriod 8 - TC Psychology"
	end

	def Button2Click(sender, e)
		@label1.Text 
	end
end

File: Pg268(FatCalc)/Pg268(FatCalc)/MainForm.py
import System.Drawing
import System.Windows.Forms

from System.Drawing import *
from System.Windows.Forms import *

class MainForm(Form):
	def __init__(self):
		self.InitializeComponent()
	
	def InitializeComponent(self):
		self._button3 = System.Windows.Forms.Button()
		self._button2 = System.Windows.Forms.Button()
		self._button1 = System.Windows.Forms.Button()
		self._label1 = System.Windows.Forms.Label()
		self._label2 = System.Windows.Forms.Label()
		self._label3 = System.Windows.Forms.Label()
		self._label4 = System.Windows.Forms.Label()
		self._label5 = System.Windows.Forms.Label()
		self._textBox1 = System.Windows.Forms.TextBox()
		self._textBox2 = System.Windows.Forms.TextBox()
		self.SuspendLayout()
		# 
		# button3
		# 
		self._button3.BackColor = System.Drawing.Color.Red
		self._button3.Location = System.Drawing.Point(187, 180)
		self._button3.Name = "button3"
		self._button3.Size = System.Drawing.Size(75, 23)
		self._button3.TabIndex = 20
		self._button3.Text = "Exit"
		self._button3.UseVisualStyleBackColor = False
		self._button3.Click += self.Button3Click
		# 
		# button2
		# 
		self._button2.BackColor = System.Drawing.Color.Yellow
		self._button2.Location = System.Drawing.Point(106, 180)
		self._button2.Name = "button2"
		self._button2.Size = System.Drawing.Size(75, 23)
		self._button2.TabIndex = 19
		self._button2.Text = "Clear"
		self._button2.UseVisualStyleBackColor = False
		self._button2.Click += self.Button2Click
		# 
		# button1
		# 
		self._button1.BackColor = System.Drawing.Color.ForestGreen
		self._button1.Location = System.Drawing.Point(25, 180)
		self._button1.Name = "button1"
		self._button1.Size = System.Drawing.Size(75, 23)
		self._button1.TabIndex = 18
		self._button1.Text = "Calc"
		self._button1.UseVisualStyleBackColor = False
		self._button1.Click += self.Button1Click
		# 
		# label1
		# 
		self._label1.BackColor = System.Drawing.SystemColors.ButtonHighlight
		self._label1.Location = System.Drawing.Point(13, 20)
		self._label1.Name = "label1"
		self._label1.Size = System.Drawing.Size(178, 23)
		self._label1.TabIndex = 21
		self._label1.Text = "Enter amount of Calories in Food"
		self._label1.TextAlign = System.Drawing.ContentAlignment.MiddleCenter
		# 
		# label2
		# 
		self._label2.BackColor = System.Drawing.SystemColors.ButtonHighlight
		self._label2.Location = System.Drawing.Point(13, 64)
		self._label2.Name = "label2"
		self._label2.Size = System.Drawing.Size(178, 23)
		self._label2.TabIndex = 22
		self._label2.Text = "Enter amount of fat grams in Food"
		self._label2.TextAlign = System.Drawing.ContentAlignment.MiddleCenter
		# 
		# label3
		# 
		self._label3.BackColor = System.Drawing.SystemColors.ButtonHighlight
		self._label3.Location = System.Drawing.Point(13, 106)
		self._label3.Name = "label3"
		self._label3.Size = System.Drawing.Size(178, 23)
		self._label3.TabIndex = 23
		self._label3.Text = "Percentage of calroies from fat"
		self._label3.TextAlign = System.Drawing.ContentAlignment.MiddleCenter
		# 
		# label4
		# 
		self._label4.BackColor = System.Drawing.SystemColors.ButtonHighlight
		self._label4.Location = System.Drawing.Point(197, 106)
		self._label4.Name = "label4"
		self._label4.Size = System.Drawing.Size(66, 23)
		self._label4.TabIndex = 24
		self._label4.TextAlign = System.Drawing.ContentAlignment.MiddleCenter
		# 
		# label5
		# 
		self._label5.BackColor = System.Drawing.SystemColors.ButtonHighlight
		self._label5.Location = System.Drawing.Point(25, 145)
		self._label5.Name = "label5"
		self._label5.Size = System.Drawing.Size(238, 23)
		self._label5.TabIndex = 25
		self._label5.TextAlign = System.Drawing.ContentAlignment.MiddleCenter
		# 
		# textBox1
		# 
		self._textBox1.Location = System.Drawing.Point(198, 22)
		self._textBox1.Name = "textBox1"
		self._textBox1.Size = System.Drawing.Size(65, 20)
		self._textBox1.TabIndex = 26
		# 
		# textBox2
		# 
		self._textBox2.Location = System.Drawing.Point(198, 64)
		self._textBox2.Name = "textBox2"
		self._textBox2.Size = System.Drawing.Size(65, 20)
		self._textBox2.TabIndex = 27
		# 
		# MainForm
		# 
		self.BackColor = System.Drawing.SystemColors.ActiveCaptionText
		self.ClientSize = System.Drawing.Size(284, 216)
		self.Controls.Add(self._textBox2)
		self.Controls.Add(self._textBox1)
		self.Controls.Add(self._label5)
		self.Controls.Add(self._label4)
		self.Controls.Add(self._label3)
		self.Controls.Add(self._label2)
		self.Controls.Add(self._label1)
		self.Controls.Add(self._button3)
		self.Controls.Add(self._button2)
		self.Controls.Add(self._button1)
		self.Name = "MainForm"
		self.Text = "Pg268(FatCalc)"
		self.ResumeLayout(False)
		self.PerformLayout()


	def Button1Click(self, sender, e):
		Cal = int(self._textBox1.Text)
		Fat = int(self._textBox2.Text)
		FatCal = float(Fat * 9) 
		Per = (FatCal/Cal)*100
		self._label4.Text = str(Per) + " %" 
		if Per < 30: 
			self._label5.Text = "Yay, this food is low in fat" 
		else:
			pass
		pass

	def Button2Click(self, sender, e):
		self._label4.Text = "" 
		self._label5.Text = ""
		self._textBox1.Text = ""
		self._textBox2.Text = ""
		
		
		pass

	def Button3Click(self, sender, e):
		Application.Exit()
		pass